File: src/api_gateway/src/properties.rs
#[derive(Default)]
pub struct Properties {
    pub shopping_list_port: i32,
    pub shopping_list_address: String,
    pub product_storage_port: i32,
    pub product_storage_address: String,
    pub recipes_port: i32,
    pub recipes_address: String,
    pub consumptions_port: i32,
    pub consumptions_address: String,
    pub notifications_port: i32,
    pub notifications_address: String,
    pub summary_port: i32,
    pub summary_address: String,
    pub api_gateway_port: i32,
    pub api_gateway_address: String,
    pub frontend_port: i32,
    pub frontend_address: String,
    pub mongodb_port: i32,
    pub mongodb_address: String,
}

pub fn get_properties() -> Properties {



    let mut x = Properties::default();
    x.api_gateway_port = 8007;
    x.shopping_list_port = 8001;
    x
}

File: src/microservices/consumptions/modules/properties.py
from jproperties import Properties


class Props:
    ShoppingListPort: int
    ProductStoragePort: int
    RecipesPort: int
    ConsumptionsPort: int
    NotificationsPort: int
    SummaryPort: int
    ApiGatewayPort: int
    FrontendPort: int
    MongoDBPort: int
    ShoppingListAddress: str
    ProductStorageAddress: str
    RecipesAddress: str
    ConsumptionsAddress: str
    NotificationsAddress: str
    SummaryAddress: str
    ApiGatewayAddress: str
    FrontendAddress: str
    MongoDBAddress: str

    def __init__(self):
        configs = Properties()
        with open('app-config.properties', 'rb') as config_file:
            configs.load(config_file)
            self.ShoppingListPort = configs.get("ShoppingListPort")
            self.ProductStoragePort = configs.get("ProductStoragePort")
            self.RecipesPort = configs.get("RecipesPort")
            self.ConsumptionsPort = configs.get("ConsumptionsPort")
            self.NotificationsPort = configs.get("NotificationsPort")
            self.SummaryPort = configs.get("SummaryPort")
            self.ApiGatewayPort = configs.get("ApiGatewayPort")
            self.FrontendPort = configs.get("FrontendPort")
            self.MongoDBPort = configs.get("MongoDBPort")
            self.ShoppingListAddress = configs.get("ShoppingListAddress")
            self.ProductStorageAddress = configs.get("ProductStorageAddress")
            self.RecipesAddress = configs.get("RecipesAddress")
            self.ConsumptionsAddress = configs.get("ConsumptionsAddress")
            self.NotificationsAddress = configs.get("NotificationsAddress")
            self.SummaryAddress = configs.get("SummaryAddress")
            self.ApiGatewayAddress = configs.get("ApiGatewayAddress")
            self.FrontendAddress = configs.get("FrontendAddress")
            self.MongoDBAddress = configs.get("MongoDBAddress")
